File: UI/DrawRectDemo/DrawRectDemo/ViewController.m
//
//  ViewController.m
//  DrawRectDemo
//
//  Created by fangyukui on 2018/4/24.
//  Copyright © 2018年 fangyukui. All rights reserved.
//

#import "ViewController.h"
#import "LiftCycle.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    
}
- (IBAction)toLiftCycle:(id)sender {
    LiftCycle *vc  =[[LiftCycle alloc]init];
    [self.navigationController pushViewController:vc animated:YES];
    
}





@end

File: 绘制布局/Draw/Draw/ViewController.swift
//
//  ViewController.swift
//  Draw
//
//  Created by fangyukui on 2018/5/30.
//  Copyright © 2018年 fangyukui. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        let containerView = ContainerView(frame: CGRect(x: 0, y: 0, width: UIScreen.main.bounds.width, height:  UIScreen.main.bounds.height))
        containerView.backgroundColor = .blue
        view = containerView
        
        let redView = RedView(frame: CGRect(x: 0, y: 0, width: 375, height: 150))

        view.addSubview(redView)
    }

    override func viewDidLayoutSubviews() {
        super.viewDidLayoutSubviews()
        print("ViewController:" + #function)
    }
    
    
    


}

